test/samples/solidity/getters_07.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.7.0;
```

Comment:
As a regression test we might want to also add an ABIEncoderV1 test. Not a blocker

Reply:
Its added in my followup PR so no need to do this here.